README.md

```diff
@@ -32,10 +32,10 @@ including tooling to build a UI in streamlit and watch your state machine execut
 
 ## Why the name Burr?
 
-Burr is named after Aaron Burr, founding father, third VP of the United States, and murderer/arch-nemesis of Alexander Hamilton.
-We imagine a world in which Burr and Hamilton lived in harmony and saw through their differences. We originally
-built Burr as a _harness_ to handle state between executions of Hamilton DAGs,
-but realized that it has a wide array of applications and decided to release it.
+Burr is named after [Aaron Burr](https://en.wikipedia.org/wiki/Aaron_Burr), founding father, third VP of the United States, and murderer/arch-nemesis of [Alexander Hamilton](https://en.wikipedia.org/wiki/Alexander_Hamilton).
+In our version of things here, we imagine a world in which Burr and Hamilton lived in harmony and saw through their differences and thus were happy to work together. We originally
```

Comment:
"In our version of things here" -- unecessary verbosity -- implied by "we imagine"